gensim/sklearn_integration/SklearnWrapperGensimLdaModel.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2011 Radim Rehurek <radimrehurek@seznam.cz>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+#
+"""
+scikit learn interface for gensim for easy use of gensim with scikit-learn
+follows on scikit learn API conventions
+"""
+from gensim import models
+from gensim import matutils
+from scipy.sparse.csr import csr_matrix
+
+
+class SklearnWrapperLdaModel(models.LdaModel,object):
+    """
+    Base LDA module
+    """
+    def __init__(self, corpus=None, num_topics=100, id2word=None,
+                 distributed=False, chunksize=2000, passes=1, update_every=1,
+                 alpha='symmetric', eta=None, decay=0.5, offset=1.0,
+                 eval_every=10, iterations=50, gamma_threshold=0.001,
+                 minimum_probability=0.01, random_state=None):
+        """
+        sklearn wrapper for LDA model. derived class for gensim.model.LdaModel
+        """
+        self.corpus = corpus
+        self.num_topics = num_topics
+        self.id2word = id2word
+        self.distributed = distributed
```

Comment:
I don't think stuff like `distributed` would really work in sklearn. Same with training or storing very large models (sklearn makes lots of deep object copies internally).